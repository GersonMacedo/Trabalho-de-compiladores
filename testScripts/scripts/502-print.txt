! -s RUN --disable-logs printer builder
! should print correctly
program p ;
begin
    print 5;
    print 42;
    print false;
    print true;
end .