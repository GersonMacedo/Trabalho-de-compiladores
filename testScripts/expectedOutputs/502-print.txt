
Performing build generation and running at testScripts/scripts/502-print.txt
No errors found
Building the program

********** TAM Interpreter (Java Version 2.1) **********
5
42
0
1

Program has halted normally.
